buildenv/jenkins/README.md

```diff
@@ -0,0 +1,194 @@
+[Eclipse OpenJ9 Jenkins Builds](https://ci.eclipse.org/openj9/)
+
+This folder contains Jenkins pipeline scripts that are used in the OpenJ9 Jenkins builds.
+
+### Triggering Pull Request Builds from Github
+
+- Current supported PR builds are "Compile", "Compile & Sanity", or "Compile & Extended"
+- Current available platforms are Linux s390x and Linux PPCLE
+- OpenJ9 committers can request builds by commenting in a pull request
+- Request a build in a PR by commenting
+    - `Jenkins compile`
+    - `Jenkins test sanity`
+    - `Jenkins test extended`
+- Request a single platform instead of "all"
+    - `Jenkins test sanity zlinux`
+    - `Jenkins test sanity plinux`
+- You can also request a "Compile & Sanity" build from an openj9-openjdk-jdk9 PR or an openj9-omr PR.
+
+##### Dependent Changes
+
+- If you have dependent change(s) in either eclipse/omr, eclipse/openj9-omr, or ibmruntimes/openj9-openjdk-jdk9, you can build & test with all needed changes
+- Request a build by including the PR ref in your trigger comment
+- Ex. Dependent change in OMR Pull Request `#123`
+    - `Jenkins test sanity depends eclipse/omr#123`
+- Ex. Dependent change in OpenJ9-OMR Pull Request `#456`
+    - `Jenkins test sanity depends eclipse/openj9-omr#456`
+- Ex. Dependent change in OpenJDK Pull Request `#789`
+    - `Jenkins test sanity depends ibmruntimes/openj9-openjdk-jdk9#789`
+- Ex. Dependent changes in OMR and OpenJDK
+    - `Jenkins test sanity depends eclipse/omr#123 ibmruntimes/openj9-openjdk-jdk9#789`
+- Ex. If you have a dependent change and only want one platform, depends comes last
+    - `Jenkins test sanity zlinux depends eclipse/omr#123`
+
+**Note:** Dependent changes can only be requested from an OpenJ9 Pull Request
+
+### Overview of Builds
+
+#### Build
+
+- Build-linux_390-64_cmprssptrs
+    - [![Build Status](https://ci.eclipse.org/openj9/buildStatus/icon?job=Build-linux_390-64_cmprssptrs)](https://ci.eclipse.org/openj9/job/Build-linux_390-64_cmprssptrs)
+    - Description:
+        - Compiles on linux_390-64_cmprssptrs
+        - Archives the SDK and test material
```

Comment:
If this is only a build job, probably no test material archived.

Reply:
The purpose here for archiving the test source is to pass both the sdk and test material to the Test Jobs. That way we don't need to reclone it in the test job and we ensure we get the same source version as the build. We could not archive it and pass a SHA, but as I said, we would have to clone again. Not sure if one method is better than the other though.

Reply:
Can you change the wording then to something like "Archives the SDK and test material for later pipeline stages"?